source/includes/Alerts/alerts.md

```diff
@@ -0,0 +1,311 @@
+# Alert
+
+## Alert Overview
+
+Alert is a notification that selected users receive on the main page of Legacy UI.
+
+## Available operations for Alert
+
+1. [List alerts](#list-all-alerts)
+2. [Create an alert](#create-an-alert)
+3. [Update an alert](#update-an-alert)
+4. [Delete an alert](#delete-an-alert)
+
+## List all alerts
+
+> Get all alerts
+
+```shell
+GET alert?v=default&includeExpired=true
+```
+
+```java
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+Request request = new Request.Builder()
+  .url("/openmrs/ws/rest/v1/alert?v=default&includeExpired=true
```

Comment:
Shouldn't parameters like `includeExpired` be explained separately? Should they be part of the default example url?

Reply:
I agree. Just updated the PR :)